patterns/filters.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Title: WooCommerce Product Filters
+ * Slug: woocommerce-blocks/product-filters
+ * Categories: WooCommerce
+ * Block Types: woocommerce/active-filters, woocommerce/price-filter, wp:woocommerce/attribute-filter, woocommerce/stock-filter
```

Comment:
Why is attribute filter block type different than the others here (i.e. having the `wp:` prefix)? Are these supposed to be the full block type references?

Reply:
That was a mistake, it was preventing the attribute filter transformation to be possible. Really good catch! 🙌 
It's fixed now.